mol_dqn/chemgraph/configs/bootstrap_dqn.json

```diff
@@ -0,0 +1,33 @@
+{
+  "atom_types": ["C", "O", "N"],
+  "max_steps_per_episode": 40,
+  "allow_removal": false,
```

Comment:
Should this be true?

Reply:
I'll leave the final decision up to you, but it seems odd to use different settings for the first step. Not allowing removal also changes the value function significantly, since if the MDP takes an action that causes all future states to have low value, it cannot revert that choice with "removal" actions.

Reply:
Yes. I agree with you the value function will be different (since the model is different), but they are still able to reach a state which is reachable with removal. Therefore the search space is the same as before. If I remember correctly, the performance of not allowing removal and allowing removal are similar (both ~0.93 without bootstrap). I can do further experiments to confirm that. I will keep it as it is if you do not strongly oppose to it.